source/fundamentals/crud/read-operations/skip.txt

```diff
@@ -0,0 +1,82 @@
+=====================
+Skip Returned Results
+=====================
+
+.. default-domain:: mongodb
+
+In this guide, we show you how to skip returned results.
+
+You can skip results using the ``skip()`` method of a 
+:java-docs:`FindIterable </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#skip(int)>`
+instance or by using the :java-docs:`Aggregates.skip() </apidocs/mongodb-driver-core/com/mongodb/client/model/Aggregates.html#sort(org.bson.conversions.Bson)>` 
+static factory method in an aggregation pipeline. 
+
+The ``skip()`` method takes an integer that specifies how many documents
+to omit from the beginning of the list of documents returned by your
+query. 
+
+You can use the ``skip()`` method of a ``FindIterable`` instance as follows:
+
+.. code-block:: java
+
+   collection.find().skip(2); 
+
+The ``Aggregates.skip()`` method implements the ``Bson`` interface to
+specify skip criteria as an argument.  
+
+You can use the ``Aggregates.sort()`` method as follows:
+
+.. code-block:: java
+
+   import com.mongodb.client.model.Aggregates;
+
+   collection.aggregate(Arrays.asList(Aggregates.skip(2)));
+   
+
+The example in this guide is about a paint store that sells eight
+different colors of paint.  The best colors sell quicker than the other
+colors. One day, a customer comes in and asks what the three
+best-selling colors are.  The following documents are in the
+``paint_inventory`` collection: 
```

Comment:
This abrupt transition and lack of explanation of "qty" is still bothering me (mentioned in https://github.com/mongodb/docs-java/pull/67#pullrequestreview-621736838 )